pytest_fixtures/component/provision_pxe.py

```diff
@@ -244,19 +245,19 @@ def pxeless_discovery_host(provisioning_host, module_discovery_sat):
     # Change host to boot from CD ISO
     Broker(
         job_template='configure-pxe-boot-rhv',
-        target_host=provisioning_host.name,
+        target_host=prov_host.name,
         target_vlan_id=settings.provisioning.vlan_id,
-        target_vm_firmware=provisioning_host._broker_args['target_vm_firmware'],
+        target_vm_firmware=provisioning_host.vm_firmware,
```

Comment:
Would suggest checking and rebase this PR over https://github.com/SatelliteQE/robottelo/pull/10273

Reply:
@Griffin-Sullivan its merged, please rebase or close this PR if you think this issue is resolved

Reply:
Rebased. Testing now.